components/FavouriteList.tsx

```diff
@@ -1,37 +1,51 @@
 import {FlatList, TouchableOpacity, View} from "react-native";
 import MuseumTile from "./MuseumTile";
-import {useEffect, useState} from "react";
-import {getFavourites, toggle} from "../async_storage/LocalStorage";
-import {useIsFocused} from "@react-navigation/native";
-
+import {useContext} from "react";
+import {FavouritesContext} from "../providers/FavouritesProvider";
+import { StyleSheet } from 'react-native';
+import useAppColorScheme from "../hooks/useAppColorScheme";
+import {darkBackground, lightBackground} from "../colors";
 
 export default function FavouriteList() {
-    const [favIds, setFavIds] = useState<Array<number>>()
-
-    useEffect(() => {
-        getFavourites().then(setFavIds)
-    }, [useIsFocused()]);
-
-
-    function onTileTap(value: number) {
-        toggle(value).then((ids) => setFavIds(ids))
-    }
+    const favouriteContext = useContext(FavouritesContext)
+    const colorScheme = useAppColorScheme();
 
     const renderItem = ({item}: { item: number }) => {
-
         return (
             <TouchableOpacity>
-                <MuseumTile id={item} onTap={() => onTileTap(item)} selected={favIds?.includes(item) ?? false}/>
+                <MuseumTile id={item} onFavouriteTap={() => favouriteContext.toggle(item)} selected={favouriteContext.favourites.includes(item) ?? false}/>
             </TouchableOpacity>
         );
     };
+
+    const containerColors = colorScheme.light ? styles.containerLight : styles.containerDark;
+
     return (
-        <View>
+        <View style={[styles.containerLayout, containerColors]}>
             <FlatList<number>
-                data={favIds}
+                style={styles.listLayout}
+                data={favouriteContext.favourites}
                 renderItem={renderItem}
                 keyExtractor={(item) => item.toString()}
             />
         </View>
     );
-}
\ No newline at end of file
+}
+
+
+
+const styles = StyleSheet.create({
+    containerLayout:{
+        height: "100%",
+    },
+    listLayout:{
+        paddingVertical: 8,
+        paddingHorizontal: 8,
```

Comment:
Let's do it simpler 😄 

```suggestion
        padding: 8,
```